core/src/main/resources/schema/infinispan-config-10.0.xsd

```diff
@@ -1802,10 +1802,7 @@
         <xs:complexType>
           <xs:attribute name="chunk-size" type="xs:int" default="512">
             <xs:annotation>
-              <xs:documentation>
-                If &gt; 0, the state will be transferred in batches of {@code chunkSize} cache entries.
-                If &lt;= 0, the state will be transferred in all at once. Not recommended. Defaults to 512.
-              </xs:documentation>
+               <xs:documentation>The number of cache entries to batch in each transfer.</xs:documentation>
```

Comment:
We should document in the upgrading guide that `chunk-size` has to be greater than 0 now.

Reply:
Done